package.json

```diff
@@ -45,7 +45,9 @@
     "typescript": "^4.9.3"
   },
   "dependencies": {
+    "@types/ramda": "^0.28.20",
     "blake-hash": "^2.0.0",
-    "ffjavascript": "^0.2.57"
+    "ffjavascript": "^0.2.57",
+    "ramda": "^0.28.0"
```

Comment:
Why do we need ramda?

Reply:
Ramda was used earlier in an util I was implementing, but now has been [removed in this commit.](https://github.com/iden3/js-crypto/pull/2/commits/26ea7d6fc3a89ee5c90cd257ec9c6f17c766c699)

Reply:
removed [here.](https://github.com/iden3/js-crypto/pull/2/commits/5b5bf5f7cb363f86ddde79ea2313b3d89bb8bf93)